source/manual/alerts/content-performance-manager-app-healthcheck-not-ok.html.md

```diff
@@ -8,17 +8,41 @@ last_reviewed_on: 2019-02-04
 review_in: 6 months
 ---
 
+## Notify Data Informed Content team
+
+Please notify Data Informed Content via Slack `#govuk-data-informed` about the alarm. The product is currently in private beta, so the development team will help solve any issues.
+
 If there is a health check error showing for Content Performance Manager, you can click on the alert to find out more details about what’s wrong. Here are the possible problems you may see:
 
 ## ETL :: no daily metrics for yesterday
 
 This means that [the ETL master process][1] that runs daily to retrieve metrics for content items has failed.  
 
-#### Notify Data Informed Content team
+To fix this problem [Re-run the master process again][1] 
 
-Please notify Data Informed Content via Slack `#govuk-data-informed` about the alarm. The product is currently in private beta, so the development team will help solve any issues.
+## ETL :: no pviews for yesterday
+
+This means that the [the ETL master process][1] that runs daily to collect metrics from Google Analytics has failed. In particular, the ETL processor that collects the core metrics.
```

Comment:
```suggestion
This means the [the ETL master process][1] that runs daily has failed to collect `pageview` metrics from Google Analytics. The issue may originate from the ELT processor responsible for collecting core metrics.
```
Also maybe link to the ETL processor for core metrics.

Reply:
Then you can remove line 27

Reply:
> Also maybe link to the ETL processor for core metrics.

I would rather not linking to codebase from the documentation as they might go out of sync.